File: clients/android-fireback/app/src/main/java/com/fireback/modules/workspaces/AppMenuEntity.java
package com.fireback.modules.workspaces;
import com.fireback.modules.workspaces.OkayResponseDto;
import com.google.gson.Gson;
import com.fireback.JsonSerializable;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;
public class AppMenuEntity extends JsonSerializable {
    public String href;
    public String icon;
    public String label;
    public String activeMatcher;
    public String applyType;
    public CapabilityEntity capability;
    public static class VM extends ViewModel {
    // upper: Href href
    private MutableLiveData< String > href = new MutableLiveData<>();
    public MutableLiveData< String > getHref() {
        return href;
    }
    public void setHref( String  v) {
        href.setValue(v);
    }
    // upper: Icon icon
    private MutableLiveData< String > icon = new MutableLiveData<>();
    public MutableLiveData< String > getIcon() {
        return icon;
    }
    public void setIcon( String  v) {
        icon.setValue(v);
    }
    // upper: Label label
    private MutableLiveData< String > label = new MutableLiveData<>();
    public MutableLiveData< String > getLabel() {
        return label;
    }
    public void setLabel( String  v) {
        label.setValue(v);
    }
    // upper: ActiveMatcher activeMatcher
    private MutableLiveData< String > activeMatcher = new MutableLiveData<>();
    public MutableLiveData< String > getActiveMatcher() {
        return activeMatcher;
    }
    public void setActiveMatcher( String  v) {
        activeMatcher.setValue(v);
    }
    // upper: ApplyType applyType
    private MutableLiveData< String > applyType = new MutableLiveData<>();
    public MutableLiveData< String > getApplyType() {
        return applyType;
    }
    public void setApplyType( String  v) {
        applyType.setValue(v);
    }
    // upper: Capability capability
    private MutableLiveData< CapabilityEntity > capability = new MutableLiveData<>();
    public MutableLiveData< CapabilityEntity > getCapability() {
        return capability;
    }
    public void setCapability( CapabilityEntity  v) {
        capability.setValue(v);
    }
    }
}

File: clients/fireback-ios/firebackios/modules/workspaces/AppMenuEntity.swift
class AppMenuEntity : Codable {
    private var _href: String?
    var `Href`: String? {
        set { _href = newValue }
        get { return _href }
    }
    private var _icon: String?
    var `Icon`: String? {
        set { _icon = newValue }
        get { return _icon }
    }
    private var _label: String?
    var `Label`: String? {
        set { _label = newValue }
        get { return _label }
    }
    private var _activeMatcher: String?
    var `ActiveMatcher`: String? {
        set { _activeMatcher = newValue }
        get { return _activeMatcher }
    }
    private var _applyType: String?
    var `ApplyType`: String? {
        set { _applyType = newValue }
        get { return _applyType }
    }
    private var _capability: CapabilityEntity?
    var `Capability`: CapabilityEntity? {
        set { _capability = newValue }
        get { return _capability }
    }
}